common/djangoapps/student/models.py

```diff
@@ -288,6 +288,8 @@ class Meta(object):
         blank=True, null=True, max_length=6, db_index=True,
         choices=LEVEL_OF_EDUCATION_CHOICES
     )
+    profession = models.TextField(blank=True, null=True)
+    specialty = models.TextField(blank=True, null=True)
```

Comment:
You don't want to add these as new fields on the UserProfile model. In production, the database table that stores the UserProfile data is very large. When you we apply the migration which adds these new columns to that table it would take a very long time to complete that operation during which read/writes to that table would be blocked.

Instead of doing this, in the past we have persisted new fields in the existing [UserProfile.meta](https://github.com/edx/edx-platform/pull/15851/files#diff-e2ffffdf6d67c9e0dee5bc10a67eb8bfR248) field. The UserProfile.meta field is a TextField that stores a serialized JSON object where the keys are the additional field names. You can take a look at what the JSON object looks like by connecting to the production LMS read replica and querying the student_userprofile table.